### PR TITLE
#1 - Project base: pom.xml, github actions

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #2 
Added `pom.xml`, settings for GitHub actions, PDD etc.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.artipie</groupId>
+    <artifactId>ppom</artifactId>
+    <version>0.3.4</version>
```
Comment (g4s8) [inline pom.xml:30]: @olegmoz we can use `0.3.6` version

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    <connection>scm:git:git://github.com/artipie/http-client.git</connection>
+    <developerConnection>scm:git:ssh://github.com:artipie/http-client.git</developerConnection>
+    <url>https://github.com/artipie/http-client/tree/master</url>
+  </scm>
+  <dependencies>
+    <dependency>
+      <groupId>com.artipie</groupId>
+      <artifactId>http</artifactId>
+      <version>0.15.2</version>
+      <scope>test</scope>
```
Comment (g4s8) [inline pom.xml:59]: @olegmoz according to the README, `http-client` provides `Slice` implementations, then `com.artipie:http` should be moved to another scope, not `test`

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <version>${junit-platform.version}</version>
+      <scope>test</scope>
+    </dependency>
```
Comment (g4s8) [inline pom.xml:83]: @olegmoz these dependencies are added automatically by parent maven profile:
```xml
      <activation>
        <file>
          <exists>${basedir}/src/test/java</exists>
        </file>
      </activation>
      <dependencies>
        <dependency>
          <groupId>org.junit.jupiter</groupId>
          <artifactId>junit-jupiter-api</artifactId>
          <scope>test</scope>
        </dependency>
        <dependency>
          <groupId>org.junit.jupiter</groupId>
          <artifactId>junit-jupiter-engine</artifactId>
          <scope>test</scope>
        </dependency>
        <dependency>
          <groupId>org.hamcrest</groupId>
          <artifactId>hamcrest</artifactId>
          <scope>test</scope>
        </dependency>
        <dependency>
          <groupId>org.mockito</groupId>
          <artifactId>mockito-core</artifactId>
          <scope>test</scope>
        </dependency>
      </dependencies>
```

**Comment by olegmoz** [comment]
@g4s8 thanks, made the changes by the comments

**Pull request merged**